.github/hex-packages.json

```diff
@@ -152,6 +152,15 @@
     "language": "elixir",
     "authorizedUsers": ["bryannaegele", "tsloughter"]
   },
+  "resource_detector_aws": {
+    "workingDirectory": "detectors/opentelemetry_resource_detector_aws",
+    "name": "Resource Detector for AWS",
+    "packageName": "opentelemetry_resource_detector_aws",
+    "tagPrefix": "opentelemetry-resource-detector-aws-v",
+    "buildTool": "rebar3",
+    "language": "erlang",
+    "authorizedUsers": ["danschultzer"]
```

Comment:
I assume I have to manage the releases for this, but lmk if either of you @bryannaegele @tsloughter should be added here.